### DISCUSSION TITLE
I would like to win Galaxy Brain

### DISCUSSION THREAD

**Discussion by PatrickDSiqueira** [Schweinepriester/github-profile-achievements#377] [Q&A] [answered]
I still haven't figured out how to get the Galaxy Brain

How does this markup of accepted answers accepted???

Could anyone help me?

**Comment by kelwinmarcelo** [comment, marked as answer]
I could even help you, but tell me, can a man be at all good? Or power corrupts the human being.

**Reply by PatrickDSiqueira** [reply]
Great thinking
